client/assets/js/meter.js

```diff
@@ -52,6 +52,30 @@ function flipSwitch(){
 }
 
 
+function switchRange(elem){
```

Comment:
Please rename to knobRange so that it is clear which UI element is being referred to. Looking at the code I am also not sure what all of the class add and removes are doing...a small comment at the top of the function may be helpful (I am guilty of this in the switch function and will add a comment to it on my next pass).

Reply:
The reason for this is since we have a switch and a knob I dont want someone to read this and then encounter issues where they think it is for the switch.